wire-ios/Wire-iOS/Sources/UserInterface/Conversation Options/ConversationGuestOptionsViewModel.swift

```diff
@@ -305,7 +332,7 @@ final class ConversationGuestOptionsViewModel {
                 switch result {
                 case .success:
                     self.updateRows()
-                    if self.link == nil && allowGuests {
+                    if self.link == nil || self.securedLink == nil && allowGuests {
```

Comment:
Does it make sense to use additional braces here to make clear how the expression is evaluated?
`(self.link == nil || self.securedLink == nil) && allowGuests`
`self.link == nil || (self.securedLink == nil && allowGuests)`

Reply:
Done! https://github.com/wireapp/wire-ios/commit/0c88209061ad06a09321480a3d8e01f5b316340c